python/cac_tripplanner/templates/partials/ios-weblinks.html

```diff
@@ -0,0 +1,5 @@
+// Keep iOS in standalone mode when following links
```

Comment:
Yeah, I was also wondering what the risk or cost of putting `ios-weblinks.html` in `base.html` would be.  Having it in 5 or 6 places is kind of a drag, and seems to add risk of a future situation in which it becomes necessary on a new template but gets forgotten (and it wouldn't be that easy/likely for us to notice...).  Is the concern that it could have unforeseen negative effects on some links so we don't want to include it anywhere we don't definitely need it?